app/controllers/application_controller.rb

```diff
@@ -1,6 +1,7 @@
 class ApplicationController < ActionController::Base
   default_form_builder GOVUKDesignSystemFormBuilder::FormBuilder
 
+  before_action :set_cache_headers
```

Comment:
is it worth adding an entry to `spec/controllers/application_controller_spec.rb` for completeness and symmetry?
I can see this line is already covered by the other request and controller specs, so it's not a coverage issue.

Reply:
Tbh I'd not realised there was an `spec/controllers/application_controller_spec` - yep, will add it